integration-tests/src/test/resources/features/AddressAPIHappyPath.feature

```diff
@@ -26,6 +26,7 @@ Feature: Address API happy path test
     #Credential Issue
     When user sends a POST request to Credential Issue end point with a valid access token
     And a valid JWT is returned in the response
+    And JWT lives for two hours
```

Comment:
Can be removed if the assertions for this line are moved to the `makeAssertions` block?

Reply:
I will address this amend in a future PR